extensions/cdc-debezium/src/main/java/com/hazelcast/jet/cdc/impl/CdcSource.java

```diff
@@ -234,26 +233,16 @@ public void setOffset(Map<String, ?> partition, Map<String, ?> offset) {
             partitionsToOffset.put(partition, offset);
         }
 
-        @Override
-        public int getFactoryId() {
-            return CdcJsonDataSerializerHook.FACTORY_ID;
-        }
-
-        @Override
-        public int getClassId() {
-            return CdcJsonDataSerializerHook.SOURCE_STATE;
-        }
-
-        @Override
-        public void writeData(ObjectDataOutput out) throws IOException {
+        void writeData(ObjectDataOutput out) throws IOException {
```

Comment:
it would be better to inline the serialization methods if you're using StreamSerializer